.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,25 @@
+#### What does this PR do?
```

Comment:
Please what are you trying to do here?

Reply:
It's the `PR template` in a `.github` I added to the root of the project. This enables the PR template to be automatically generated when anyone raises a PR. The essence is you don't get to type in the various heading of the PR template.